scripts/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Base image
+FROM ubuntu
+
+# Update the sources list
+RUN apt-get update
+
+# Install packages
+RUN apt-get install -y tar git curl nano wget dialog net-tools build-essential python python-dev python-distribute python-pip zlib1g-dev python-virtualenv apache2 libapache2-mod-wsgi
+
+# Enable wsgi module
+RUN a2enmod wsgi
+
+# Create cache directories
+RUN mkdir /var/cache/apache2/python-egg-cache && \
+    chown www-data:www-data /var/cache/apache2/python-egg-cache/
+
+# Set up GA4GH server
+RUN mkdir /srv/ga4gh
+WORKDIR /srv/ga4gh
+RUN virtualenv ga4gh-server-env
```

Comment:
Interesting... I guess we should play with both options and see what works better for us.